docs/eks.md

```diff
@@ -0,0 +1,170 @@
+
+2. Deploy the Operator [using](https://kubernetes.io/docs/reference/using-api/server-side-apply/)
+    the following command:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply --server-side -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/bundle.yaml -n postgres-operator
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        customresourcedefinition.apiextensions.k8s.io/perconapgbackups.pg.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/perconapgclusters.pg.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/perconapgrestores.pg.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/postgresclusters.postgres-operator.crunchydata.com serverside-applied
+        serviceaccount/percona-postgresql-operator serverside-applied
+        role.rbac.authorization.k8s.io/percona-postgresql-operator serverside-applied
+        rolebinding.rbac.authorization.k8s.io/service-account-percona-postgresql-operator serverside-applied
+        deployment.apps/percona-postgresql-operator serverside-applied
+        ```
+
+    As the result you will have the Operator Pod up and running.
+
+2. The operator has been started, and you can deploy your Percona Distribution
+    for PostgreSQL cluster:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/cr.yaml -n postgres-operator
+    ```
+
+    ??? example "Expected output"
+
+        ``` {.text .no-copy}
+        erconapgcluster.pg.percona.com/cluster1 created
+        ```
+
+    !!! note
+
+        This deploys default Percona Distribution for PostgreSQL configuration.
+        Please see [deploy/cr.yaml](https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/cr.yaml)
+        and [Custom Resource Options](operator.md) for the configuration
+        options. You can clone the repository with all manifests and source code
+        by executing the following command:
+
+        ```{.bash data-prompt="$" }
+        $ git clone -b v{{ release }} https://github.com/percona/percona-postgresql-operator
+        ```
+
+        After editing the needed options, apply your modified `deploy/cr.yaml`
+        file as follows:
+
+        ```{.bash data-prompt="$" }
+        $ kubectl apply -f deploy/cr.yaml -n postgres-operator
+        ```
+
+    The creation process may take some time. The process is over when both
+    Operator and replica set Pods have reached their Running status:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl get pg
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        NAME       ENDPOINT                     STATUS   POSTGRES   PGBOUNCER   AGE
+        cluster1   cluster1-pgbouncer.pgo.svc   ready    3          3           143m
```

Comment:
I think in your case it will be: 
```suggestion
        cluster1   cluster1-pgbouncer.postgres-operator.svc   ready    3          3           143m
```

pgo it is namespace, but you use postgres-operator